lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/passportCri/ContractTest.java

```diff
@@ -210,6 +231,20 @@ public RequestResponsePact validRequestReturnsIssuedCredential(PactDslWithProvid
                 .toPact();
     }
 
+    @Pact(provider = "PassportCriProvider", consumer = "IpvCoreBack")
+    public RequestResponsePact invalidRequestReturnsError(PactDslWithProvider builder)
+            throws Exception {
+        return builder.given("dummyApiKey is a valid api key")
+                .given("dummyAccessToken is an invalid access token")
+                .uponReceiving("Invalid POST request")
+                .path("/credential")
+                .method("POST")
+                .headers("x-api-key", PRIVATE_API_KEY, "Authorization", "Bearer dummyAccessToken")
+                .willRespondWith()
+                .status(404)
```

Comment:
I don't see why this would return a 404, it's the same request as above.
Remember that this request will be sent to the real API code in the provider test, so if you expect a 404 back then the request should be for a URL will actually return a 404.

Reply:
Ah, I missed that `dummyAccessToken` is invalid for this test. Maybe name is `dummyInvalidAccessToken` so it's more obvious?

Reply:
Maybe also name the pact so it's more obvious e.g. `invalidAccessTokenReturns404`